library/mongodb/dashboards/mongodb.json

```diff
@@ -0,0 +1,286 @@
+                    },
+                    "layout": {
+                        "column": 5,
+                        "row": 7,
+                        "height": 3,
+                        "width": 4
+                    },
+                    "title": "Bytes Out Per Second",
+                    "rawConfiguration": {
+                        "nrqlQueries": [
+                            {
+                                "accountId": 0,
+                                "query": "SELECT average(`network.bytesOutPerSecond`) AS `Bytes Out` FROM MongosSample FACET entityName TIMESERIES 1 MINUTE"
+                            }
+                        ]
+                    }
+                },
+                {
+                    "visualization": {
+                        "id": "viz.line"
+                    },
+                    "layout": {
+                        "column": 9,
+                        "row": 7,
+                        "height": 3,
+                        "width": 4
+                    },
+                    "title": "Requests Per Second",
+                    "rawConfiguration": {
+                        "nrqlQueries": [
+                            {
+                                "accountId": 0,
+                                "query": "SELECT average(`network.requestsPerSecond`) FROM MongosSample FACET entityName TIMESERIES 1 MINUTE"
+                            }
+                        ]
+                    }
+                },
+                {
+                    "visualization": {
+                        "id": "viz.table"
+                    },
+                    "layout": {
+                        "column": 1,
+                        "row": 10,
+                        "height": 3,
+                        "width": 8
+                    },
+                    "title": "Database Sizes",
+                    "rawConfiguration": {
+                        "nrqlQueries": [
+                            {
+                                "accountId": 0,
+                                "query": "SELECT latest(`stats.dataInBytes`) AS `Size in Bytes`, latest(`stats.objects`) AS `Objects` FROM MongoDatabaseSample FACET entityName LIMIT 1000"
+                            }
+                        ]
+                    }
+                },
+                {
+                    "visualization": {
+                        "id": "viz.bar"
+                    },
+                    "layout": {
+                        "column": 9,
+                        "row": 10,
+                        "height": 3,
+                        "width": 4
+                    },
+                    "title": "Available Connections",
+                    "rawConfiguration": {
+                        "nrqlQueries": [
+                            {
+                                "accountId": 0,
+                                "query": "SELECT latest(`connections.available`) FROM MongodSample FACET entityName"
```

Comment:
@kidk regarding this query I had to change from:

`"query": "SELECT latest('connections.available') FROM MongosSample FACET entityName"`

to

`"query": "SELECT latest('connections.available') FROM MongodSample FACET entityName"`

and the chart appears :)